data/sles4sap/qe_sap_deployment/trento_azure.yaml

```diff
@@ -1,5 +1,5 @@
 provider: "%PROVIDER%"
-apiver: 1
+apiver: 2
```

Comment:
This one is about (but does not needs) https://github.com/SUSE/qe-sap-deployment/pull/79